### ISSUE TITLE
Public key inclusion for Trustchain community

### ISSUE THREAD

**Issue by synctext** [Tribler/tribler#2978]
now we just include the hash of public keys, instead of simply including the full public key.

**Comment by qstokkink** [comment]
MultiChainBlocks are created with the public key, see: https://github.com/Tribler/tribler/blob/devel/Tribler/community/multichain/community.py#L157. They are then sent as-is over the network, see: https://github.com/Tribler/tribler/blob/devel/Tribler/community/multichain/block.py#L317.

You can see this in action in, for example, the unit tests https://github.com/Tribler/tribler/blob/1af7c6f60d663399d3bd1979285de9c68e36ad35/Tribler/Test/Community/Multichain/test_multichain_block.py#L25. If you inspect your local database, you will also find that it contains the full public keys.

**tl;dr** We are already sending the full public key.

**Comment by synctext** [comment]
ahhh, good to hear. thnx.
this came up with the clean-slate crawler.
